src/PositionManager.sol

```diff
@@ -186,6 +186,9 @@ contract PositionManager is
         } else if (action == Actions.SWEEP) {
             (Currency currency, address to) = params.decodeCurrencyAndAddress();
             _sweep(currency, _mapRecipient(to));
+        } else if (action == Actions.TAKE) {
+            (Currency currency, address recipient, uint256 amount) = params.decodeCurrencyAddressAndUint256();
+            _take(currency, _mapRecipient(recipient), _mapTakeAmount(amount, currency));
```

Comment:
hmm `params.decodeCurrencyAddressAndUint256()` isnt tested in `CalldataDecoder.t.sol` 🧐 

can you add?

Reply:
added